src/main/java/control/ControllerManager.java

```diff
@@ -0,0 +1,29 @@
+package control;
+
+import gui.RootPane;
+import lombok.Getter;
+
+/**
+ * Class wrapper for model management controllers.
+ * @author alex
+ */
+public class ControllerManager {
```

Comment:
It's meant to be a builder/wrapper for the controllers, to keep a modicum of separation